universum/lib/module_arguments.py

```diff
@@ -135,6 +135,15 @@ def _needs_default_parser(subparsers_action, args):
         return True
 
     def _add_default_parser(self, args):
+        """
+        We need to manually add the default parser because of the following reasons.
+        Argparser allows calls like this: ``./executable --arg=value1 subcommand --arg=value2``
+        after processing this is transformed to two different namespaces, each of which have
+        argument 'arg' with two different values. So calling ``universum -vt=p4 poll`` and
```

Comment:
According to my test, the "transformed to two different namespaces, each of which have argument 'arg' with two different values" statement is wrong.

```
$ ./test.py --arg value1 subparser --arg value2
Namespace(arg='value2')
```